### PR TITLE
Option for Sequence of losses (for models with multiple outputs)

### PR DESCRIPTION
**Pull request by timdebruin**
Keras / Tensorflow allows for lists of losses when a model has multiple outputs. This change reflects this option in the Experiment loss field's type hint.

### CODE REVIEW

File: zookeeper/tf/experiment.py [hunk 1]
```diff
@@ -21,5 +21,7 @@ class Experiment:
     # Parameters
     epochs: int = Field()
     batch_size: int = Field()
-    loss: Optional[Union[keras.losses.Loss, str]] = Field()
+    loss: Optional[
+        Union[Sequence[Union[keras.losses.Loss, str]]], Union[keras.losses.Loss, str]
```
Comment (AdamHillier) [inline zookeeper/tf/experiment.py:25]:
```suggestion
        Union[Sequence[Union[keras.losses.Loss, str]], keras.losses.Loss, str]
```
